main.js

```diff
@@ -11,3 +11,14 @@ for (var i = 0; i < chars.length; i++) {
 
 $phrase.querySelector('span').classList.add('current')
 var $currentChar = document.querySelector('.current')
+
+document.addEventListener('keypress', function(event) {
+    if (event.key === $currentChar.textContent) {
+      console.log('correct')
+      $currentChar.style = 'color: #1e90ff; font-size: 19px'
```

Comment:
better to set a class that will have some styles associated with it. You can use `.classList.add` and `.classList.remove`.